lib/std/monotimes.nim

```diff
@@ -53,18 +57,14 @@ when defined(macosx):
 
 when defined(js):
   proc getJsTicks: float =
-    ## Returns ticks in the unit seconds
-    {.emit: """
-      var isNode = typeof module !== 'undefined' && module.exports
-
-      if (isNode) {
-        var process = require('process');
-        var time = process.hrtime()
-        return time[0] + time[1] / 1000000000;
-      } else {
-        return window.performance.now() / 1000;
-      }
-    """.}
+    ## Returns ticks in the unit seconds.
+    when defined(nodejs):
+      {.emit: """let process = require('process');
```

Comment:
now in nep1 :)
https://nim-lang.github.io/Nim/nep1.html
> Prefer multiline triple quote literals to start with a newline

Reply:
ok, but the generated codes are ugly

Reply:
indentation is wrong either way in generated code; for generated code would may pipe result to a beautifier in future work (similar to https://github.com/nim-lang/Nim/pull/10554 but for js)